pyrobosim/pyrobosim/gui/main.py

```diff
@@ -306,3 +311,11 @@ def on_close_click(self):
             print(f"[{robot.name}] Closing {robot.location}")
             self.canvas.close_location(robot)
             self.update_button_state()
+
+    def on_toggle_click(self):
+        """Callback to toggle collision polygons."""
+        robot = self.get_current_robot()
+        if robot and robot.location:
+            print(f"[{robot.name}] Toggling collision polygons")
```

Comment:
No need to check a robot or print its name, it's a global option. So you can just ignore this check.

Also, might be nice if depending on the toggle state it printed "Enabling" vs. "Disabling" for example